### ISSUE TITLE
Windows 10 2004 > Search not opening Outlook elements

### ISSUE THREAD

**Issue by RalfRoemling** [Open-Shell/Open-Shell-Menu#478]
Updated yesterday from Win10.1909 to Win10.2004

When searching within the Start-Button-Menu elements from Outlook 2010 are shown but when trying to open them they don't. Files are doin' fine.

Can anybody confirm this? Is there a solution I eventually missed?

Windows 10 19041.508 
Outlook 2010 14.0.7258.5000 (32bit)
Openshell 4.4.152

**Comment by bonzibudd** [comment]
It looks like this is the case for me as well.

Mine looks a little different though. For some reason, the Outlook results don't show up at all, until I click "See more results." But the results that show up there are not clickable, nor right-clickable.

Are you able to go into Right-click->Properties? If so, a screenshot may be useful so we can see the file type, location, and possibly more info.

**Comment by RalfRoemling** [comment]
Yep, I get the Right-Click-menu but nothing happens when clicking _Open_ or _Answer_. Strange thing is when selecting _Open Conversation_ it seems OpenShell is being restarted

**Comment by bonzibudd** [comment]
That is strange, it sounds like it must have come from a change in Windows, or maybe Outlook. I'll have to take a look in a fresh installation and see if I can troubleshoot this.

**Comment by bonzibudd** [comment]
I just realized. There was an issue from over a year ago with the same problem. It is #186. In that case, this is definitely not a one-off case, so maybe it needs to be prioritized by the Open-Shell team. 

In the meantime, I would go into the "Search Box" settings in Open-Shell, and see if any of the options there happen to fix the issue.

**Comment by RalfRoemling** [comment]
Hi Anything new on this one? I can't find any setting in "Search Box" that seems corresponding with this kind of behavior.

**Comment by RalfRoemling** [comment]
@bonzibudd havn't heard anything on this one and it's really a show stopper for me. Maybe you or @ge0rdi may find time have a look? I mean hey, #186 is from 2019 now :-) Thx a lot in advance, Ralf

**Comment by RalfRoemling** [comment]
Just noticed sth very weired: 
- While this doesn't work on my desktop machine on my notebook it does. 
- Thing is both were setup afresh with Win10.21h2 and the same Officepackage 2010 Plus
- Only difference was on the NB Thunderbird wasn't installed

I than had the supicion, it may have to do sth with the "Standard Program Settings" and bingo. When resetting these to Windows default and reselecting them again OpenShell is working as intended.

I don't know what exactly has to be checked, but maybe this'll get you an idea where to lookl.

**Comment by blackcrack** [comment]
>
> Openshell 4.4.152

>
>I'll have to take a look in a fresh installation and see if I can troubleshoot this.

maaan, take the newest :
https://ci.appveyor.com/project/passionate-coder/open-shell-menu/branch/master/artifacts

check it 's out
and give reply how it's works ..

@bonzibudd , and kick they in the ... if they's needed \*g*..

best
Blacky

**Comment by passingby5302** [comment]
For me it was HKCR regedit TypeLib issues, due to having tried multiple versions of Office.

My key was correct here, but I had other Office versions' in Typelib that I needed to delete.  I only kept the 9.4 keys, as I only used Outlook 2010.

https://social.technet.microsoft.com/Forums/en-US/ed8df373-8ba4-4305-bb5e-cffef7536e04/windows-7-search-with-outlook-2010-results-does-not-open

Eg of keys to fix:
HKEY_CLASSES_ROOT\TypeLib\{00062FFF-0000-0000-C000-000000000046}\9.4\0\win64\


After fixing these, my Outlook 2010 search items opened liked normal on Windows 7 x64 and Windows 10 x64 21h1 machines!  Took months of searching to fix.
